src/main/java/edu/ucsb/cs56/ucsb_courses_search/controller/advice/AuthControllerAdvice.java

```diff
@@ -20,13 +20,19 @@ public boolean getIsLoggedIn(OAuth2AuthenticationToken token){
     @ModelAttribute("id")
     public String getUid(OAuth2AuthenticationToken token){
         if (token == null) return "";
-        return token.getPrincipal().getAttributes().get("id").toString();
+        for (java.util.Map.Entry<java.lang.String, java.lang.Object> entry : token.getPrincipal().getAttributes().entrySet()) {
+            System.out.println(entry.getKey() + ":" + entry.getValue().toString());
+        }
+        return token.getPrincipal().getAttributes().get("sub").toString();
     }
 
     @ModelAttribute("login")
     public String getLogin(OAuth2AuthenticationToken token){
         if (token == null) return "";
-        return token.getPrincipal().getAttributes().get("login").toString();
+
+        //return token.getPrincipal().getAttributes().get("login").toString();
+	return token.getPrincipal().getAttributes().get("email").toString();
```

Comment:
This should probably return the user's name instead of their email.

Also, tab this to keep the formatting consistent, please. Thanks!